connectors/slack/install.js

```diff
@@ -18,40 +19,47 @@ slack_client_id=${ctx.body.configuration.slack_client_id}
 # OAuth Client Secret of the Slack application
 slack_client_secret=${ctx.body.configuration.slack_client_secret}
 
-# OAuth Scopes to request
+# Signing Secret of the Slack application
+slack_signing_secret=${ctx.body.configuration.slack_signing_secret}
+
+# OAuth Bot user token scopes to request
 slack_scope=${ctx.body.configuration.slack_scope || ''}
 
+# OAuth User token scopes to request
+slack_user_scope=${ctx.body.configuration.slack_user_scope || ''}
+
 # Allowed returnTo URLs. Comma delimited. Use wildcard (*) at the end of a URL to allow for a prefix match
 fusebit_allowed_return_to=${ctx.body.configuration.fusebit_allowed_return_to}
 `,
-        nodejs: {
-            files: {
-                'package.json': {
-                    engines: {
-                        node: "10"
-                    },
-                    dependencies: {
-                        "@fusebit/add-on-sdk": "^0.0.1",
-                        "superagent": "^5.2.2"
-                    }
-                },
-                'index.js': getTemplateFile('index.js'),
-                'initial.html': getTemplateFile('initial.html'),
-                'configure.js': getTemplateFile('configure.js'),
-            }
+    nodejs: {
+      files: {
+        'package.json': {
+          engines: {
+            node: '10',
+          },
+          dependencies: {
+            '@fusebit/add-on-sdk': '^1.3.1',
+            superagent: '^5.2.2',
+            '@slack/events-api': '^2.3.4',
+          },
         },
-        metadata: {
-            fusebit: {
-                editor: {
-                    navigationPanel: {
-                        hideFiles: [],
-                    }
-                }
-            },
-            ...ctx.body.metadata
-        }
-    });
-    
-    return { status: 200, body: { status: 200 }};
-};
+        'index.js': getTemplateFile('index.js'),
+        'storage.js': getTemplateFile('storage.js'),
+        'initial.html': getTemplateFile('initial.html'),
+        'configure.js': getTemplateFile('configure.js'),
+      },
+    },
+    metadata: {
+      fusebit: {
+        editor: {
+          navigationPanel: {
+            hideFiles: [],
+          },
+        },
+      },
+      ...ctx.body.metadata,
```

Comment:
Ditto about URL of template manager added by the template manager itself.

Reply:
Works with portal, further template contract standardization is tracked by fivequarters/q5#737. Nothing to do here.